packages/application-shell/src/components/navbar/menu-items.styles.ts

```diff
@@ -102,6 +107,7 @@ const listStyles = css`
   flex-direction: column;
   min-height: 0;
   flex: 1 1 0;
+  --safe-start: 100% 100%;
```

Comment:
I wonder if we need to define it here instead of relying on the initial state of `percentageX` and `percentageY`? And if that initial value is needed at all?

Reply:
updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3536/commits/b662e8f9258af8523279fa2a301a35959e6a6b33